File: katas/02-sort_inner_content/gabriel.js
function sortTheInnerContent(words)
{
  let words_array = [];
  for (let i=0; i<words.split(' ').length; i++){
    let word = words.split(' ')[i]
    if (word.length!=1){
        words_array.push(word[0] + 
        word.substring(1, word.length-1).split('').sort().reverse().join('') + 
        word[word.length-1]);
    } else {
      words_array.push(word);
    }
  }
  return words_array.join(' ');
}

File: katas/02-sort_inner_content/gabriel.py
def sort_the_inner_content(words):
  new_string = ""
  for word in words.split():
      if len(word) != 1:
          re_order = list(word[1:-1])
          re_order.sort(reverse=True)
          new_word = word[0] + ''.join(re_order) + word[-1]
      else:
          new_word = word
      new_string += new_word + " "
  return new_string[:-1]

